src/instruments/src/ND/pages/ArcMode.tsx

```diff
@@ -105,10 +102,18 @@ export const ArcMode: React.FC<ArcModeProps> = ({ symbols, adirsAlign, rangeSett
                             || fmaLatMode === LateralMode.TRACK) && !fmaLatArmed) || !flightPlanManager.getCurrentFlightPlan().length) && (
                             <TrackLine x={384} y={620} heading={heading} track={track} />
                         )}
+                        <g clipPath="url(#arc-mode-tcas-clip)">
```

Comment:
TCAS symbols should also not be clipped by the map clip mask. TCAS has it's own clip mask that is correctly applied here.